Document environment variables (#2)
openstudiocoalition/openstudiocoalition.github.io: markdown_source/docs/reference/openstudio_application_interface.md

[![BCL Window](img/os_interface/bcl_window.png "Click to view")](img/os_interface/bcl_window.png)

*Above: This window gives you access to the online BCL to download measures or components.*

------

## Environment Variables

> A variety of environment variables can be used to modify the OpenStudio Applicaton's operation.  Environment variables can be set on the command line for a specific session or stored permantently in the advanced system settings on Windows or the ~/.bashrc file on Unix and Mac.

Reviewer: ```suggestion
A variety of environment variables can be used to modify the OpenStudio Application's operation.  Environment variables can be set on the command line for a specific session or stored permanently in the advanced system settings on Windows or in your profile file (eg `~/.bashrc`) on Unix.
```

### OpenStudio Application Environment Variables

- `OPENSTUDIO_APPLICATION_DEBUG`, if set then debug level logging is enabled
- `OPENSTUDIO_APPLICATION_LOGFILE_PATH`, if set to a path then logging information is sent to the file path
- `OPENSTUDIO_APPLICATION_SLEEP_AT_START`, if set to an integer the application sleeps that many seconds before starting to allow a debugger to attach

> - `OPENSTUDIO_APPLICATION_START_TAB_INDEX`, if set to an integer the application opens the specified tab first { SITE = 0, SCHEDULES = 1, CONSTRUCTIONS = 2, LOADS = 3, SPACE_TYPES = 4, GEOMETRY = 5, FACILITY = 6, SPACES = 7, THERMAL_ZONES = 8, HVAC_SYSTEMS = 9, OUTPUT_VARIABLES = 10, SIMULATION_SETTINGS = 11, RUBY_SCRIPTS = 12, RUN_SIMULATION = 13, RESULTS_SUMMARY = 14 }

Reviewer: I'm a bit worried about maintainability here, perhaps we should add a note saying your can reference the most up to date by looking at the enum VerticalTabID in OSDocument.hpp?

https://github.com/openstudiocoalition/OpenStudioApplication/blob/139696fd6ff217fd3bfac08aaba1de23d157e25d/src/openstudio_lib/OSDocument.hpp#L142-L159
